components/base/AlertBar.tsx

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+import styled from "@emotion/styled";
+import infoIcon from "../../public/images/info-icon-grey.svg";
+import warningIcon from "../../public/images/warning-icon-yellow.svg";
+import {
+  GREY_1,
+  GREY_2,
+  API_STATUS_BAR_YELLOW,
+  PARAGRAPH_TEXT,
+} from "../colors";
+import { ROBOTO } from "../typography";
+
+const StyledAlertBar = styled.div`
+  display: flex;
```

Comment:
What are your thoughts on adding `align-items: center;` here in order to vertically center the icon if the text wraps to more than one line? I'm going back and forth

Reply:
Yeah, I was going back and forth on that too! 

I tried going off of this [page](https://projects.invisionapp.com/d/main#/console/19172953/400928381/preview) in the mock-ups that has the icon positioned at the top when the text wraps.  Thought it looked better that way too. Let's ask the team about it. 👍

Reply:
Looks like the verdict is to keep it positioned at the top.

Reply:
Sounds good! Thanks for talking through that with me/the team 💯

Reply:
😎👍